### PR TITLE
Bring in nextflow files from 2023-rehgt-nextflow

### PR DESCRIPTION
**Pull request by taylorreiter**
...which will be archived once this is merged (or deleted...the history isn't that rich)

All tests are passing on the test data set locally.

**I chose to skip the following files**:
* README.md & CITATIONS.md (they will be a part of a documentation PR later)
* LICENSE (same as the one already in the repo, MIT)
* template.png (picture of the template, not necessary)
* envs/ (already in the repo)
* github/ISSUE_TEMPLATE.md and github/PULL_REQUEST_TEMPLATE.MD (these felt unnecessary)

**I also removed all references to 2023-rehgt-nextflow and updated it to 2023-rehgt** and made scripts executables so they'll actually run. Lastly, there was a missed script in #13, so I added that.

**Everything was already reviewed over at 2023-rehgt-nextflow**, so the review should focus on whether I should add the files I chose to omit, or any other high level issues you identify here.

See #17 for upcoming PRs.

### CODE REVIEW

File: workflows/rehgt.nf [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                    hmmscan_hgt_candidates.out.tblout)
+}
+
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    COMPLETION EMAIL AND SUMMARY
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+workflow.onComplete {
```
Comment (mertcelebi) [inline workflows/rehgt.nf:103]: This will not work really. It should be the version from seqqc. We should port that to the nextflow template

Comment (mertcelebi) [reply workflows/rehgt.nf:103]: Alternatively you can remove this and all the related email cruft. Like line 105 includes `multiqc_report` and that's def not relevant

**Review by mertcelebi — APPROVED** [review]
Approving with the suggestion to remove the email cruft!

For the README, I'd make sure to include a word or two about our deployment on Tower with wave.

File: workflows/rehgt.nf (hunk 1 above)
Comment (taylorreiter) [reply workflows/rehgt.nf:103]: so remove this and `assets/email*` for now, with an issue to address in future PRs if we want it?

**Comment by taylorreiter** [comment]
> For the README, I'd make sure to include a word or two about our deployment on Tower with wave.

ok! do you mind if I put this in an issue for now and address it during my documentation PR?

File: workflows/rehgt.nf (hunk 1 above)
Comment (taylorreiter) [reply workflows/rehgt.nf:103]: See #29

**Pull request merged**